models/user.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
```

Comment:
I think we can remove this file. I don't think we need a user on the client side. And I don't think you need direct access to the database with Mongoose.